.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Obviously this `.babelrc` file is used just for transpiling local files... not the application itself.